pysixtrack/be_beamfields/gaussian_fields.py

```diff
@@ -75,7 +80,8 @@ def _get_transv_field_gauss_ellip(sigmax, sigmay, Delta_x, Delta_y, x, y, mathli
         w_etaBE_re, w_etaBE_im = wfun(etaBE_re / S, etaBE_im / S)
 
         expBE = exp(
-            -aby * aby / (2 * sigmay * sigmay) - abx * abx / (2 * sigmax * sigmax)
+            -aby * aby / (2 * sigmay * sigmay)
+            - abx * abx / (2 * sigmax * sigmax)
```

Comment:
Just out of curiosity: why is the leading - in line 83 treated differently than the one in line 84 by the linter?